### PR TITLE
IMU_GYRO_RATEMAX set system default to 400 Hz

### PR DESCRIPTION
**Pull request by dagar**
- MC default is still 800 Hz

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rc.mc_defaults [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 set VEHICLE_TYPE mc
 
+param set IMU_GYRO_RATEMAX 800
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/rc.mc_defaults:10]:
```suggestion
param set-default IMU_GYRO_RATEMAX 800
```

**Comment by dagar** [comment]
Case by case for specific MC airframes might be better.

**Comment by dagar** [comment]
> * MC default is still 800 Hz

Although I'm wondering if post https://github.com/PX4/PX4-Autopilot/pull/15820 400 Hz is a better conservative default for most multicopters as well.

**Comment by bkueng** [comment]
Can't we afford 800?

**Comment by dagar** [comment]
> Can't we afford 800?

Yes we can generally afford it, I'm just trying to preemptively optimize overall to improve the outcome for all the random combinations of configuration users stumble into. Obviously it's quite variable depending on the board, but very roughly we're talking about 10-15% cpu. I believe only relatively small (possibly medium-small depending on your perspective) multicopters will benefit beyond 400 Hz, although it could be interesting to actually quantify this for larger vehicles (https://github.com/betaflight/betaflight/issues/7327#issuecomment-452994940).

There are other minor side benefits to the lower rate for things like `gyro_fft`, `logger`, and even real time performance (significantly less context switching), but mainly it's that I only want to pay for something when it's beneficial.

**Comment by bkueng** [comment]
I'm ok with reducing it, it's just that it's another param you need to change to get best performance (at least for smaller vehicles), and it's potentially easier to miss performance regressions (even better would be to check that in CI).
We could set the default for the racer boards to 1/2khz.

**Comment by LorenzMeier** [comment]
The key thing is to summarize this on a settings page that is called "Performance" so that users see what they have chosen.

**Comment by dagar** [comment]
> I'm ok with reducing it, it's just that it's another param you need to change to get best performance (at least for smaller vehicles), and it's potentially easier to miss performance regressions (even better would be to check that in CI).
> We could set the default for the racer boards to 1/2khz.

This is only the actual underlying default value, and MC is still defaulting to 800 Hz. From here I want to specialize across boards and probably have a few more generic quadcopter options that roughly correspond to size classes without being too specific.

**Review by bkueng — APPROVED** [review]
> This is only the actual underlying default value, and MC is still defaulting to 800 Hz

Ok, didn't see that

**Pull request merged**